spec/lib/action_kit/helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe ActionKit::Helper do
+  before do
+    ENV['AK_PASSWORD'] = 'ak_password'
+    ENV['AK_USERNAME'] = 'ak_username'
```

Comment:
Would echo that this is something that bit us earlier this week and we might want to try to find a more elegant solution to handling this.